system/lzma/LzmaEnc.cpp

```diff
@@ -1923,7 +1923,7 @@ static SRes LzmaEnc_CodeOneBlock(CLzmaEnc *p, Bool useLimits, UInt32 maxPackSize
 static SRes LzmaEnc_Alloc(CLzmaEnc *p, UInt32 keepWindowSize, ISzAlloc *alloc, ISzAlloc *allocBig)
 {
   UInt32 beforeSize = kNumOpts;
-  Bool btMode;
+  Bool btMode [[maybe_unused]];
```

Comment:
Alternatively I could move this into the #ifdef on line 1930.  Would probably be better.

Reply:
Better as

```
  #ifdef COMPRESS_MF_MT
  Bool btMode;
  btMode = (p->matchFinderBase.btMode != 0);
  p->mtMode = (p->multiThread && !p->fastMode && btMode);
  #endif

```